### PR TITLE
Colors

### PR DESCRIPTION
**Pull request by herrethan**
Getting rid of color palette in favor of theme overrides!
https://reonomy.atlassian.net/browse/SG-34

Now that MUI supports custom colors within the theme, we can utilize this for all our liminal custom colors. Say goodbye to `ReonomyPalette` for good.

New color palette:
image

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -17,7 +16,7 @@ function ReonomyStyles({ children }: StylesProps) {
   );
 }
 
-export { ReonomyPalette, ReonomyCssBaseline, ReonomyTheme };
+export { ReonomyCssBaseline, ReonomyTheme };
```
Comment (ironprice91) [inline src/index.tsx:19]: I noticed that `ReonomyPalette` is being used in visage. Shouldn't we still export ReonomyPalette here?

Here's an example of where it's being used in visage: `visage/frontend/src/components/ownership-feedback/email/styles.ts`

**Review by ironprice91 — COMMENTED** [review]
Looks good, I had a question on `ReonomyPalette` exporting

**Review by ironprice91 — APPROVED** [review]

File: src/index.tsx [hunk 2]
```diff
@@ -17,7 +16,7 @@ function ReonomyStyles({ children }: StylesProps) {
   );
 }
 
-export { ReonomyPalette, ReonomyCssBaseline, ReonomyTheme };
```
Comment (ironprice91) [inline src/index.tsx:20]: Would there be a follow-up visage PR after `ReonomyPalette` is removed?

**Review by herrethan — COMMENTED** [review]
@ironprice91 yeah i have a visage PR that removes that, and does the full color refactor whole-hog. since these are all breaking changes.

File: src/theme.tsx [hunk 3]
```diff
@@ -63,6 +107,17 @@ const ReonomyTheme = createMuiTheme({
...
+    labels: {
+      '0': '#959595', // grey
+      '1': '#f2d600', // yellow
+      '2': '#e79f27', // orange
+      '3': '#d36027', // red
+      '4': '#00a074', // green
+      '5': '#cc7aa8', // pink
+      '6': '#5fb2e6', // light blue
+      '7': '#0773b3', // dark blue
+      '8': '#3a3b4d' // black
```
Comment (jdents) [inline src/theme.tsx:119]: Do these need to be number strings or can they be named?

File: src/theme.tsx [hunk 4]
```diff
@@ -1,11 +1,36 @@
...
+    tertiary: SimplePaletteColorOptions;
+    accent: SimplePaletteColorOptions;
+    labels: Labels;
+  }
+  interface PaletteOptions {
+    tertiary?: SimplePaletteColorOptions;
+    accent?: SimplePaletteColorOptions;
+    labels?: Labels;
+  }
+}
```
Comment (jdents) [inline src/theme.tsx:32]: 👍

**Review by jdents — APPROVED** [review]
👍

**Review by adarsh-murthy — APPROVED** [review]

File: src/theme.tsx (hunk 3 above)
Comment (herrethan) [reply src/theme.tsx:119]: These can be whatever we want them to be, I believe. I figured we'd keep with our 0-8 label color convention for now. I think Joel has some future plans to clean up those colors too.

**Pull request merged**